docs/devel/storage.md

```diff
@@ -31,7 +36,54 @@ This will create
 
 After a few moments the hostpath-provisioner pod should be running, we are now ready to create PVCs against it. Any requested PVCs will be stored in `/var/run/kubevirt/hostpath/<pvc-name>-<hash>`. The PV retention policy will be `delete`, which means that when you delete the PVC, the matching PV is also removed and the directory created on the host is removed as well.
 
-## Deploy CDI controller
+# Local volume based storage
+
+Local volume is in Beta since Kubernetes 1.10 and thus no longer requires you to enable it through the [feature-gates](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) parameter of the kubelet. To enable local volume for the development environment please follow the following steps. Note local volumes WILL work on multiple nodes due to the fact that PVCs don't get bound until a POD actually uses the PVC, at which point the local volume is bound on the node the pod is running on. The following steps are based on information found in the [Local Persistent Storage User Guide](https://github.com/kubernetes-incubator/external-storage/blob/master/local-volume/README.md)
+
+## Preliminary steps
+
+Since disks or volumes cannot be added to a node after a container is started (nodes are running in containers in the development environment). The mount source will have to be some directories that exist in the container. This can be achieved by using the --bind option when mounting. This example is creating 3 mount disks, but its possible to add or remove as many as one wants, as long as there is at least 1.
+
+Make some directory based mount points:
```

Comment:
Just to make sure that you are aware of it, these changes will not survive. As soon as you stop the cluster these changes are lost.

Reply:
Right, and also the ConfigMap/SA/Cluster Roles/etc will be lost as well. which is expected. But seeing the discussion below, it might not matter if we internalize the local volume stuff into kubevirtci. So lets focus on that, and then decide what we can remove from here.